tests/create.pp

```diff
@@ -83,3 +83,19 @@
     port     => 80,
   }],
 }
+
+rds_instance { 'db-name-5':
```

Comment:
Even just creating the instance would be fine. It would be great to have the connection example though as well.